src/terraform-resource/in/in.go

```diff
@@ -208,6 +221,34 @@ func (r Runner) writeBackendStateToFile(envName string, client terraform.Client)
 	return ioutil.WriteFile(stateFilePath, stateContents, 0777)
 }
 
+func (r Runner) writeJSONPlanToFile(envName string, client terraform.Client) error {
+
+	stateContents, err := client.StatePull(envName)
```

Comment:
Replace this with `client.Output` and remove `models.TfState` struct definition. Should be simpler and won't be affected if the statefile schema ever changes.

Reply:
I might be wrong but could you please check your comment on that one: https://github.com/ljfranklin/terraform-resource/pull/114#discussion_r409925685 ? I originally used the `client.Output`. I believe this implementation is more clean than the one before. The schema might change some day but this is also applies for `client.Output`. The test will fail in such a case. What do you think?

Reply:
The issue with the original implementation was that the plan output was getting written to `response.metadata` which appears in the Concourse UI. I'm saying update `writeJSONPlanToFile` to call `client.Output(envName)` and remove `StatePull` and JSON marshaling calls.